apps/extension/src/routes/popup/paths.ts

```diff
@@ -1,4 +1,11 @@
 export enum PopupPath {
   INDEX = '/',
   LOGIN = '/login',
+  SETTINGS = '/settings',
+  SETTINGS_INDEX = '/settings/',
```

Comment:
Why have two? We should get rid of one

Reply:
Why did I have two lol? Is this required? If not, let's get rid of both.

Reply:
it`s not required)